spec.mediawiki

```diff
@@ -487,6 +585,127 @@ reconnect or to act at the appropriate time is considered a violation of
 [[#Rules_of_Community_Conduct|community conduct rules]], so can result in 
 penalties.
 
+===Order Book Subscriptions===
+
+An order book can be viewed and tracked by subscribing to a market. 
+
+'''JSON-RPC method:''' <code>orderbook</code>, '''originator: ''' client
+
+{|
+! field       !! type   !! description
+|-
+| basemarket  || string || currency code for the market's base asset
+|-
+| quotemarket || string || currency code for the market's quote asset
+|}
+
+The response will contain the complete market order book. 
+The order book and all updates include a '''sequence ID''', which increments by 
++1 for every valid order the DEX receives. The client is responsible for 
+tracking the sequence ID to ensure all order updates are received. If an update
+appears to be missing, the client should resubscribe to the market to 
+synchronize the order book from scratch.
+
+'''Response'''
+
+{|
+! field       !! type   !! description
+|-
+| book  || object || An OrderBook object
+|-
+| seq   || int    || A sequence ID
+|}
+
+'''JSON OrderBook object'''
+
+{|
+! field  !! type !! description
+|-
+| bookid || int  || an order book ID, included with future updates
```

Comment:
What does this come from?  Is it a unique response ID or is it a global order book state that might potentially be shared with other clients subscribed to these messages?

Reply:
I guess its more of a market ID. Just so that the full "DCR"/"BTC" strings wouldn't need to be sent with every update. Probably just a 1 or 2 digit integer. Would `marketid` sound better?

Reply:
Nah, it actually makes sense as bookid, I just wasn't clear on whether the param was a static key or something that incremented or otherwise changed

Reply:
damn. I already changed it to `marketid` and I kind of like it like that.

Reply:
lol, that's cool too.